src/cursor.coffee

```diff
@@ -641,15 +641,20 @@ class Cursor extends Model
   # * `options` (optional) {Object} with the following keys:
   #   * `includeNonWordCharacters` A {Boolean} indicating whether to include
   #     non-word characters in the regex. (default: true)
+  #   * `includeEol` A {Boolean} indicating whether to include the end of the
```

Comment:
Can we call the option `includeEOL`? We standardized on all caps for initialisms for 1.0.